vault-upgrade-hook/src/main/java/biz/netcentric/vlt/upgrade/handler/userpreferences/UserPreferencesHandler.java

```diff
@@ -12,9 +12,11 @@
 import javax.jcr.Value;
 
 import org.apache.jackrabbit.vault.packaging.InstallContext;
+import org.apache.sling.api.resource.ResourceResolver;
```

Comment:
This class was working only with JCR before to make it not dependant on Sling, I would prefer to have no sling dependency here.

Reply:
From "Requirements" section of the main README I've understood that Sling is required and working with high-level API (Sling) is much more comfortable rather than with low-level (JCR). If we aim of JCR-only systems - ok, will remove sling deps here.